docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -580,59 +776,115 @@ public int Shape
 
     ![Stacks](media/stacks.png)
 
-<a name="textvisual"></a>
-## Text Visual
 
-The text visual renders text within a control.
 
-**Figure: Text visual**
+**Usage:**
+
+The following example illustrates how to draw a BevelledCube.
+   ```csharp
+   PrimitiveVisual _primitiveVisual = new PrimitiveVisual();
+   _primitiveVisual.Shape = PrimitiveVisualShapeType.BevelledCube;
+   _primitiveVisual.MixColor = new Vector4(0.4f, 0.4f, 1.0f, 1.0f);
+   // Two initial coord. indicate the top-left corner; the third one indicates the shift toward the observer
+   _primitiveVisual.LightPosition = new Vector3(0, 0, 1000);
+   _primitiveVisual.ScaleDimensions = new Vector3(1.0f, 0.5f, 1.4f);
+   _primitiveVisual.BevelPercentage = 0.5f;
+   _primitiveVisual.BevelSmoothness = 0.0f;
+
+   _visualView.AddVisual("BevelledCube", _primitiveVisual);
+
+   ```
+
+
+**Figure: PrimitiveVisual from the above code**
+
+![Primitive visual](media/visuals/PrimitiveVisualExample_BevelledCube.png)
+
+
+
+<!---------------------------------------------------------------------------------------------------------------------------->
+<a name="textVisual"></a>
+### TextVisual
 
-![Text visual](media/HelloWorld.png)
+The TextVisual renders text within a control.
```

Comment:
```suggestion
 The `TextVisual` renders text within a control. 
```
Global: please specify the control.

Reply:
https://github.com/Samsung/tizen-docs/pull/1367#discussion_r540655300